lib/workers/branch/schedule.ts

```diff
@@ -123,13 +168,22 @@ export function isScheduledNow(config: RenovateConfig): boolean {
 
   // We run if any schedule matches
   const isWithinSchedule = configSchedule.some((scheduleText) => {
-    const massagedText = scheduleMappings[scheduleText] || scheduleText;
-    const parsedSchedule = later.parse.text(fixShortHours(massagedText));
-    logger.debug({ parsedSchedule }, `Checking schedule "${scheduleText}"`);
+    if (scheduleText.indexOf(minutesChar) === 0) {
```

Comment:
Didn't we validated before, that this is a valid schedule?, so i think this condition can be removed.

Reply:
I think this is just about checking "is this a cron schedule or a later one?"

Reply:
I think we can just try to parse it, as the validation above will do. it will return `undefined` for non cron strings, so then it must be a `later` string.

Reply:
OK, @olegkrivtsov please do a full parse here.